pkg/pool-manager/pod_pool.go

```diff
@@ -42,6 +42,11 @@ func (p *PoolManager) AllocatePodMac(pod *corev1.Pod) error {
 		return nil
 	}
 
+	// allocate only when the network status is no exist
```

Comment:
could you explain why you do it?

"if the network status annotation is already set, it means *???*"

Reply:
the pod namespace is already exist because multus runs and create the network status label in the pod